index.user.js

```diff
@@ -369,11 +417,28 @@ class BlackholeAttack extends ProjectileAttack {
     getAttackName() {
         return "blackhole";
     }
+    shouldAttack(delta, enemies) {
+        if(enemies.length < 3) {
+            return false;
+        }
+        return CanAttack(this.getAttackName());
+    } 
+    attack(x, y) {
+        SetMouse(START_POS - k_nDamagePointx, (APP.renderer.height / 2) + 100);
+        AttackManager().m_mapKeyCodeToAttacks.get(this.getAttackData().keycode)();
+    }        
 }
 class MeteorAttack extends ProjectileAttack {
     getAttackName() {
         return "boulder";
     }
+    process(enemies) {
+        this.attack();
+    }
+    attack() {
+        SetMouse(k_nDamagePointx + 50,  (APP.renderer.height / 2) + 100);
```

Comment:
Where does this 50 come from? Is it the size of the Meteor sprite?

Reply:
@TehDmitry ping

Reply:
k_nDamagePointx + 50 — is just little bit right from left side of screen